app/reports/related_resource_types.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Invoke via:
+# bin/rails r -e production "RelatedResourceTypes.report" > related_resource_types.csv
+class RelatedResourceTypes
+  # path within description
+  JSON_PATH = '$.relatedResource[*].type'
```

Comment:
I'm not sure we've been consistent with this, but I wonder if we should make this a ["strict"](https://www.postgresql.org/docs/current/functions-json.html#FUNCTIONS-SQLJSON-STRICT-AND-LAX-MODES) JSON path

```suggestion
  JSON_PATH = 'strict $.relatedResource[*].type'
```

I haven't internalized when we need to make this distinction and when we don't, but I do vaguely recall bogus results in reports early on before we discovered that `lax` was the default for JSON path expressions in Postgres.

Reply:
Since it's an example, we may want to toss it in there so we don't get bitten with weird lax data bugs in the future. It may or may not be valuable to re-run the changed report in an env like QA to verify the prior results.

Reply:
Agree this is a good approach and updated the report to use strict with a pointer to the docs.